src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -470,6 +470,27 @@ private void OnProcessExit(object sender, EventArgs e)
 
         // Synchronization primitive for controlling initialization of Default load context
         private static readonly object s_initLock = new Object();
+
+        // Occurs when an Assembly is loaded
+        public static event AssemblyLoadEventHandler AssemblyLoad 
+        { 
```

Comment:
Why did we choose to add these to ALC as opposed to introduce the AppDomainAugments?